src/core/a-node.js

```diff
@@ -32,19 +33,13 @@ class ANode extends HTMLElement {
     this.mixinEls = [];
   }
 
-  onReadyStateChange () {
-    if (document.readyState === 'complete') {
-      this.doConnectedCallback();
-    }
-  }
-
   connectedCallback () {
     // Defer if DOM is not ready.
-    if (document.readyState !== 'complete') {
-      document.addEventListener('readystatechange', this.onReadyStateChange.bind(this));
+    if (!ready.isReady) {
```

Comment:
should be `isDOMReady` but not sure because isReady involves more than DOM now

Reply:
Yeah, it now encompasses both DOM (= all elements are parsed) and A-Frame (everything registered, incl. third-party components/systems/etc...) being ready. Not sure what an apt name would be. 

Perhaps the name could be changed to not reflect what needs to be ready, but that this action can take place. So something like `canInitializeElements`?

Reply:
`isAnodeReadyToLoad`? too specific?

Reply:
Feels a tad specific, but it also reads as if "Anode" _is ready_, instead of everything being ready "for Anode" to load. And we'd probably want to avoid using "load" here as the `doConnectedCallback` implementations do more than just invoke `load`.